OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -282,6 +282,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface ITemporaryBlocker
+	{
+		bool CanRemoveBlockage(Actor self, Actor blocking);
```

Comment:
@Phrohdoh: `blocking` refers the actor that is blocked (i.e. the unit that wants to go through the gate), not another blocking actor actor that `self` may be able to remove.